salt/modules/ps.py

```diff
@@ -772,3 +772,32 @@ def psaux(name):
     ret = []
     ret.extend([sanitize_name, found_infos, pid_count])
     return ret
+
+
+def status(filter):
+    """
+    Returns a list of processes according to their states.
+    See https://psutil.readthedocs.io/en/latest/index.html\
+?highlight=status#process-status-constants
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' ps.status [running, idle, stopped, sleeping, dead, \
+zombie, ...]
+    """
+    ret = []
+    if filter is not None and filter != "":
+        sanitize_name = str(filter)
```

Comment:
**suggestion**  `status` would be a better name
```suggestion
      status = str(filter)
```

It would also need to be changed below.

Reply:
Done !